src/os/bluestore/BlueStore.h

```diff
@@ -513,8 +513,9 @@ class BlueStore : public ObjectStore,
       ++nref;
     }
     void put() {
-      if (--nref == 0)
-	delete this;
+      if (--nref == 0) {
+	//delete this;
+      }
```

Comment:
Shouldn't we do blob (partial?) cleanup on putting into the pool. Otherwise it might waste a huge amount of space while being on the bench...